### PR TITLE
Pass locale into business endpoint for localization

### LINKED ISSUE #38: Business endpoint missing support for locale params
Looks like [`client.business`](https://github.com/Yelp/yelp-ruby/blob/develop/lib/yelp/endpoint/business.rb#L23) doesn't support locale params as expected by readme and API documentation.

### PR DESCRIPTION
**Pull request by tomelm**
fixes #38

### CODE REVIEW

File: spec/yelp/endpoint/business_spec.rb [hunk 1]
```diff
@@ -18,6 +19,18 @@
     its('business.name') { should eql('Yelp') }
```
Comment (mittonk) [inline spec/yelp/endpoint/business_spec.rb]: Check "yelp.com" here?

File: spec/yelp/endpoint/business_spec.rb [hunk 2]
```diff
@@ -6,6 +6,7 @@
   let(:api_keys) { real_api_keys }
   let(:business) { 'yelp-san-francisco' }
   let(:client) { Yelp::Client.new(api_keys) }
+  let(:locale) { Hash[lang: 'fr'] }
```
Comment (mittonk) [inline spec/yelp/endpoint/business_spec.rb]: `fr` is a nice language, but it's not a locale --- that'd be `fr_FR`, `fr_CA`, or similar.

File: lib/yelp/endpoint/business.rb [hunk 3]
```diff
@@ -34,8 +34,8 @@ def business(id)
       #
       # @param id [String, Integer] the business id
       # @return [Faraday::Response] the raw response back from the connection
-      def business_request(id)
-        result = @client.connection.get PATH + id
+      def business_request(id, locale = {})
+        result = @client.connection.get PATH + id, locale
```
Comment (mittonk) [inline lib/yelp/endpoint/business.rb]: `@param` for `lang` or `locale`?

File: spec/yelp/endpoint/business_spec.rb (hunk 2 above)
Comment (tomelm) [inline spec/yelp/endpoint/business_spec.rb]: The API doesn't officially support a locale but we do call that group of params "locale parameters" so I followed in pattern

File: lib/yelp/endpoint/business.rb (hunk 3 above)
Comment (mittonk) [inline lib/yelp/endpoint/business.rb]: (That is, please update the docstring)

File: spec/yelp/endpoint/business_spec.rb (hunk 2 above)
Comment (mittonk) [inline spec/yelp/endpoint/business_spec.rb]: OK, I get it.

Perhaps `locale_args` would be clearer in that case?

Comment (tomelm) [inline spec/yelp/endpoint/business_spec.rb]: Given `Search` uses `locale`, do we want to stick with `locale` for consistency-sake or switch over.

Comment (mittonk) [inline spec/yelp/endpoint/business_spec.rb]: Great, that consistency is a stronger requirement than my whims.

**Comment by mittonk** [comment]
:shipit:

**Pull request merged**